Userland/Libraries/LibSanitizer/UBSanitizer.cpp

```diff
@@ -15,50 +15,71 @@ Atomic<bool> AK::UBSanitizer::g_ubsan_is_deadly;
     warnln(fmt, ##__VA_ARGS__);    \
     dbgln("\x1B[31m" fmt "\x1B[0m", ##__VA_ARGS__);
 
+#define ABORT_ALWAYS()                                            \
+    do {                                                          \
+        WARNLN_AND_DBGLN("UBSAN: This error is not recoverable"); \
+        abort();                                                  \
```

Comment:
hmm, I thought `abort()` would also cause a backtrace to be created, but apparently not :yakfused: